pythonforandroid/recipes/libzbar/__init__.py

```diff
@@ -8,13 +8,14 @@
 
 class LibZBarRecipe(Recipe):
 
-    version = '0.10'
+    version = '0.23.1'
 
-    url = 'https://github.com/ZBar/ZBar/archive/{version}.zip'
+    url = 'https://github.com/mchehab/zbar/archive/{version}.tar.gz'
```

Comment:
Why is that? It seems to be a less popular and "implicit" fork

Reply:
hey, I don't have time to address most of these things right now, but https://github.com/ZBar/ZBar hasn't been updated since 2012.  https://github.com/mchehab/zbar is actively maintained.

Reply:
Thanks for the info, do we know if it's the new "official fork", is the API compatible with old version?

Reply:
The API seems compatible (I didn't have to change any code to switch to it).  I don't see any evidence that the original maintainer knows it exists, but it's the version released in gentoo and mentioned on wikipedia.  It seems to be part of linuxtv.org .

ZBar 0.10 from 2012 had some bugs with binary data for me, so I use newer versions.

Reply:
Indeed, good point for Gentoo, I'm sold :smile: 
https://packages.gentoo.org/packages/media-gfx/zbar
I think it deserves a comment for this URL explaining we are using this fork on purpose and don't want to roll back unless we have good reason. Because sometimes some recipes point to the wrong URL by mistake and someone may think it's the case for this one

Reply:
Good plan.